zulipterminal/model.py

```diff
@@ -334,6 +337,9 @@ def set_search_narrow(self, search_query: str) -> None:
         self.unset_search_narrow()
         self.narrow.append(["search", search_query])
 
+    def get_alert_words(self) -> List[str]:
+        return self._alert_words.copy()
+
```

Comment:
I didn't comment on this before, but this function placement in the file seems strange too - between two related functions?

Reply:
will fix it asap

Reply:
Note that strictly this *could* go into a separate commit with a test to eg. ensure that one can't modify the data externally, but this is fine.